plugins/gradle/src/org/jetbrains/plugins/gradle/internal/daemon/DaemonAction.java

```diff
@@ -28,7 +32,22 @@ protected ServiceRegistry getDaemonServices(DaemonClientFactory daemonClientFact
     if (myServiceDirectoryPath != null && !myServiceDirectoryPath.isEmpty()) {
       layout.setGradleUserHomeDir(new File(myServiceDirectoryPath));
     }
-    DaemonParameters daemonParameters = new DaemonParameters(layout);
+    DaemonParameters daemonParameters = null;
+    boolean isGradle5Dot3OrNewer = GradleVersion.current().compareTo(GradleVersion.version("5.3")) >= 0;
```

Comment:
minor: better to use base version for comparison to handle RC versions also.
`GradleVersion.current().getBaseVersion().compareTo(GradleVersion.version("5.3")) >= 0;`

Reply:
Thanks for catching it. Done.